tests/writer.js

```diff
@@ -24,7 +24,8 @@ require([
                     onReady: function () {
                         done();
                     }
-                }
+                },
+                features: {'experimental_ui': false},
```

Comment:
Hmm, there are a lot of these. What do you think of turning experimental UI off by default for tests? Maybe that would be tricky?